traffic_ops/app/db/create_tables.sql

```diff
@@ -143,6 +143,42 @@ LANGUAGE plpgsql;
 
 ALTER FUNCTION public.on_delete_current_timestamp_last_updated() OWNER TO traffic_ops;
 
+--
+-- Name: update_ds_timestamp_on_insert(); Type: FUNCTION; Schema: public; Owner: traffic_ops
+--
+
+CREATE OR REPLACE FUNCTION update_ds_timestamp_on_insert()
+    RETURNS trigger
+    AS $$
+BEGIN
+    UPDATE deliveryservice
+    SET last_updated=now()
+    WHERE id IN (
+        SELECT deliveryservice
+        FROM CAST(NEW AS deliveryservice_tls_version)
+    );
+    RETURN NULL;
```

Comment:
This block was also indented with tabs previously, converting it spaces was an improvement

Reply:
real indentation > spaces

but the rest of `create_tables.sql` is fake-indented, so it's consistent.